.travis.yml

```diff
@@ -26,6 +26,22 @@ matrix:
     - python: 3.6
       env:
         - IPYTHON=ipython[all]
+    - python: 3.6
+      dist: xenial
+      env:
+        - IPYTHON=ipython[all]
+    #- python: 3.7
+    #  dist: xenial
+    #  env:
+    #    - IPYTHON=ipython[all]
+    #    - NUMPY_VERSION=1.14.6
+    #- python: 3.8
+    #  dist: xenial
+    #  env:
+    #    - IPYTHON=ipython[all]
+    #    - NUMPY_VERSION=1.19.0
```

Comment:
please remove commented code if not needed

Reply:
We are going to have to run on Python 3.7 and 3.8 to be credible.

However, given that we cannot yet figure out how to run on Python 3.6 on Xenial, let’s keep Python 3.7 and 3.8 commented out until we figure that out.

Reply:
WDYT @maxpumperla @marload ?

Reply:
I'm not an expert on this, but could we change to another dist that works for 3.7 and 3.8? @cclauss

Reply:
bump @marload @maxpumperla @cclauss